src/babel/transformation/transformers/es7/function-bind.js

```diff
@@ -0,0 +1,41 @@
+// https://github.com/zenparsing/es-function-bind
+
+import * as t from "../../../types";
+
+export var metadata = {
+  optional: true,
+  stage: 0
+};
+
+function getTempId(scope) {
```

Comment:
I think this UID caching might introduce race conditions with nested binding expressions.